test/fast/Listing paths/Running "nvm which 0.0.2" should display only version 0.0.2.

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 
 \. ../../../nvm.sh
+\. ../../../common.sh
```

Comment:
there's a number of places where this path is incorrect; `common.sh` is not a sibling of `nvm.sh`, it's inside `test`

Reply:
Oops! Sorry I missed that, staring at all those dots can get confusing sometimes 😆